### ISSUE TITLE
GifImagePlugin.getheader claims to return list; actually returns tuple

### ISSUE THREAD

**Issue by cancan101** [python-pillow/Pillow#1082]
`GifImagePlugin.getheader` [claims to return list](https://github.com/python-pillow/Pillow/blob/42b59046e7daaa80283255cb9f94e1894e56d8ca/PIL/GifImagePlugin.py#L410) but [actually returns tuple](https://github.com/python-pillow/Pillow/blob/42b59046e7daaa80283255cb9f94e1894e56d8ca/PIL/GifImagePlugin.py#L492).

this causes problems in the [gifmaker](https://github.com/python-pillow/Pillow/blob/54173d2c67bc8ed8953b56e67d4719676f43a073/Scripts/gifmaker.py#L81) script:

```
<ipython-input-21-9400b2d06cd2> in compress(infile, outfile)
     79     seq = image_sequence(im)
     80 
---> 81     makedelta(fp, seq)
     82 
     83     fp.close()

<ipython-input-21-9400b2d06cd2> in makedelta(fp, sequence)
     37 
     38             # global header
---> 39             for s in getheader(im) + getdata(im):
     40                 fp.write(s)
     41 

TypeError: can only concatenate tuple (not "list") to tuple
```

**Comment by wiredfool** [comment]
So, it looks like sometime around 2.1.0 the return value changed from a list to a tuple of (list, list|None), with the original data in the first element of the tuple. The comment wasn't updated at that point, nor was gifmaker, which hasn't been seriously updated since the fork. I suspect that this would be the correct patch:

```
            # global header
            for s in getheader(im)[0] + getdata(im):
                fp.write(s)
```

**Comment by aclark4life** [comment]
Status please? I can't tell if this is fixed or not. Thanks

**Comment by radarhere** [comment]
The fix suggested here was merged in #1134. Removing the fix and testing, I can reproduce the original bug, so I would say this is fixed.

**Comment by hugovk** [comment]
@radarhere Thanks for checking. Closing this issue.
